pkg/kubelet/images/image_manager_test.go

```diff
@@ -231,12 +264,12 @@ func TestSerializedPuller(t *testing.T) {
 
 	useSerializedEnv := true
 	for _, c := range cases {
-		puller, fakeClock, fakeRuntime, container := pullerTestEnv(c, useSerializedEnv, nil)
+		puller, fakeClock, fakeRuntime, container := pullerTestEnv(t, c, useSerializedEnv, nil)
 
 		t.Run(c.testName, func(t *testing.T) {
 			ctx := context.Background()
 			for _, expected := range c.expected {
-				fakeRuntime.CalledFunctions = nil
+				fakeRuntime.CalledFunctions = []string{}
```

Comment:
My suggestion remains the same: use `fakeRuntime.ClearCalls()` as it clears not only `CalledFuncitons` but also other fields. it will cause some test cases to fail as they may depend on previous test case results. Another suggestion is to run test cases in parallel.

Reply:
btw, `ClearCalls` is not used and can be deleted.

Reply:
Sorry I did read what you said in other request.  I was trying to use it but I got lots of test failures.  Haven't had a chance to really dig into that now so I left it out of PR.

Reply:
no prob. Please, remove ClearCalls and leave `fakeRuntime.CalledFunctions = nil` intact, thanks.

Reply:
did both now!